merge_knownexploited is expected to be a procedure using PostgreSQL

**Describe the bug**
While running version 8.0.0 using PostgreSQL the maven plugin fails due to `merge_knownexploited` being a function instead of a procedure. This seems to be a bug for PostgreSQL only because `merge_knownexploited` is declared as a procedure for Oracle, MySQL and others.

**Version of dependency-check used**
The problem occurs using version 8.0.0 of the maven plugin.

**Log file**
```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:8.0.0:check (default-cli) on project product-parent: Fatal exception(s) analyzing product-parent: One or more exceptions occurred during analysis:
[ERROR] 	UpdateException: java.sql.BatchUpdateException: Batch entry 0 CALL merge_knownexploited('CVE-2021-27104','Accellion','FTA','Accellion FTA OS Command Injection Vulnerability','2021-11-03','Accellion FTA 9_12_370 and earlier is affected by OS command execution via a crafted POST request to various admin endpoints.','Apply updates per vendor instructions.','2021-11-17',NULL) was aborted: ERROR: merge_knownexploited(character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying) is not a procedure
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create an empty PostgreSQL database.
2. Add missing to `initialize_postgres.sql` inside of `merge_knownexploited`, see additional context, and execute on newly created database.
3. Run dependency check maven goal.

**Expected behavior**
The plugin successfully executes updates and checks using PostgreSQL.

**Additional context**
`initialize_postgres.sql` also seems to be missing a semicolon inside `merge_knownexploited` which prevents the migration file from succeeding initially.

I recreated the database wir the updated script but the problem still occures:
```
[ERROR] 	UpdateException: java.sql.BatchUpdateException: Batch entry 0 CALL merge_knownexploited('CVE-2021-27104','Accellion','FTA','Accellion FTA OS Command Injection Vulnerability','2021-11-03','Accellion FTA 9_12_370 and earlier is affected by OS command execution via a crafted POST request to various admin endpoints.','Apply updates per vendor instructions.','2021-11-17',NULL) was aborted: ERROR: merge_knownexploited(character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying) is not a procedure
[ERROR]   Hinweis: To call a function, use SELECT.
```

My guess is that https://github.com/jeremylong/DependencyCheck/blob/0961742cc4d69f291b8a543d56b8b47aa81d300a/core/src/main/resources/data/dbStatements_postgresql.properties#L21 should be `SELECT` instead of `CALL`.

--- a/core/src/main/resources/data/initialize_mssql.sql
+++ b/core/src/main/resources/data/initialize_mssql.sql
@@ -26,8 +26,8 @@ if exists (SELECT 1 FROM sysobjects WHERE name='insert_software' AND xtype='P')
     drop procedure insert_software;    
 if exists (SELECT 1 FROM sysobjects WHERE name='knownExploited' AND xtype='U')
     drop table knownExploited;
-if exists (SELECT 1 FROM sysobjects WHERE name='merge_knownexpoited' AND xtype='P')
-    drop procedure merge_knownexpoited;    
+if exists (SELECT 1 FROM sysobjects WHERE name='merge_knownexploited' AND xtype='P')
+    drop procedure merge_knownexploited;    
 
 CREATE TABLE vulnerability (id int identity(1,1) PRIMARY KEY, cve VARCHAR(20) UNIQUE,
 	description VARCHAR(8000), v2Severity VARCHAR(20), v2ExploitabilityScore DECIMAL(3,1), 
@@ -235,12 +235,12 @@ IF EXISTS(SELECT * FROM knownExploited WHERE cveID=@cveID)
     SET vendorProject=@vendorProject, product=@product, vulnerabilityName=@vulnerabilityName, 
         dateAdded=@dateAdded, shortDescription=@shortDescription, requiredAction=@requiredAction, 
         dueDate=@dueDate, notes=@notes
-    WHERE cveID=@cveID
+    WHERE cveID=@cveID;
 ELSE
     INSERT INTO knownExploited (vendorProject, product, vulnerabilityName,
         dateAdded, shortDescription, requiredAction, dueDate, notes, cveID) 
     VALUES (@vendorProject, @product, @vulnerabilityName,
-        @dateAdded, @shortDescription, @requiredAction, @dueDate, @notes, @cveID)
+        @dateAdded, @shortDescription, @requiredAction, @dueDate, @notes, @cveID);
 END;
 
 GO

--- a/core/src/main/resources/data/initialize_mysql.sql
+++ b/core/src/main/resources/data/initialize_mysql.sql
@@ -19,7 +19,7 @@ DROP PROCEDURE IF EXISTS dependencycheck.cleanup_orphans;
 DROP PROCEDURE IF EXISTS dependencycheck.update_vulnerability;
 DROP PROCEDURE IF EXISTS dependencycheck.insert_software;
 DROP PROCEDURE IF EXISTS dependencycheck.merge_ecosystem;
-DROP PROCEDURE IF EXISTS dependencycheck.merge_knownexpoited;
+DROP PROCEDURE IF EXISTS dependencycheck.merge_knownexploited;
 DROP TABLE IF EXISTS software;
 DROP TABLE IF EXISTS cpeEntry;
 DROP TABLE IF EXISTS `reference`;
@@ -307,7 +307,7 @@ INSERT INTO knownExploited (`cveID`, `vendorProject`, `product`, `vulnerabilityN
 END //
 DELIMITER ;
 
-GRANT EXECUTE ON PROCEDURE dependencycheck.merge_knownexpoited TO 'dcuser';
+GRANT EXECUTE ON PROCEDURE dependencycheck.merge_knownexploited TO 'dcuser';
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON dependencycheck.* TO 'dcuser';
 

--- a/core/src/main/resources/data/initialize_oracle.sql
+++ b/core/src/main/resources/data/initialize_oracle.sql
@@ -238,7 +238,7 @@ EXCEPTION
 END;
 /
 
-GRANT EXECUTE ON merge_knownexpoited TO dcuser;
+GRANT EXECUTE ON merge_knownexploited TO dcuser;
 
 CREATE OR REPLACE PROCEDURE update_vulnerability(p_cveId IN vulnerability.cve%type,
                                       p_description IN vulnerability.description%type,

--- a/core/src/main/resources/data/initialize_postgres.sql
+++ b/core/src/main/resources/data/initialize_postgres.sql
@@ -6,7 +6,7 @@ DROP FUNCTION IF EXISTS public.save_property;
 DROP FUNCTION IF EXISTS public.update_vulnerability;
 DROP FUNCTION IF EXISTS public.insert_software;
 DROP FUNCTION IF EXISTS public.merge_ecosystem;
-DROP FUNCTION IF EXISTS public.merge_knownexpoited;
+DROP FUNCTION IF EXISTS public.merge_knownexploited;
 DROP TABLE IF EXISTS software;
 DROP TABLE IF EXISTS cpeEntry;
 DROP TABLE IF EXISTS reference;
@@ -123,7 +123,7 @@ IF EXISTS(SELECT 1 FROM knownExploited WHERE cveID=p_cveID) THEN
     SET vendorProject=p_vendorProject, product=p_product, vulnerabilityName=p_vulnerabilityName, 
         dateAdded=p_dateAdded, shortDescription=p_shortDescription, requiredAction=p_requiredAction, 
         dueDate=p_dueDate, notes=p_notes
-    WHERE cveID=p_cveID
+    WHERE cveID=p_cveID;
 ELSE
     INSERT INTO knownExploited (cveID, vendorProject, product, vulnerabilityName,
             dateAdded, shortDescription, requiredAction, dueDate, notes)
@@ -133,7 +133,7 @@ END IF;
 END
 $$ LANGUAGE plpgsql;
 
-GRANT EXECUTE ON FUNCTION public.merge_knownexpoited(varchar(20), VARCHAR(255), VARCHAR(255), VARCHAR(500), CHAR(10), VARCHAR(2000), VARCHAR(1000), CHAR(10), VARCHAR(2000)) TO dcuser;
+GRANT EXECUTE ON FUNCTION public.merge_knownexploited(varchar(20), VARCHAR(255), VARCHAR(255), VARCHAR(500), CHAR(10), VARCHAR(2000), VARCHAR(1000), CHAR(10), VARCHAR(2000)) TO dcuser;
 
 CREATE FUNCTION update_vulnerability (
     IN p_cveId VARCHAR(20), IN p_description VARCHAR(8000), IN p_v2Severity VARCHAR(20), 

